skrub/_table_vectorizer.py

```diff
@@ -678,6 +783,11 @@ def fit_transform(self, X: ArrayLike, y: ArrayLike = None) -> ArrayLike:
         if self.verbose:
             print(f"[TableVectorizer] Assigned transformers: {self.transformers}")
 
+        # split the univariate transformers on each column
+        # to be able to parallelize the encoding
+        if self.n_jobs not in (None, 1):
```

Comment:
Maybe this condition could be exported to a method? That would make this more readable and expandable in the future.
Something like that:
```py
    @property
    def is_parallelized(self) -> bool:
        return self.n_jobs not in (None, 1)
```
and you would do
```suggestion
        if self.is_parallelized:
```